lms/lmsweb/views.py

```diff
@@ -130,6 +130,16 @@ def favicon():
     )
 
 
+@webapp.before_request
+def banned_page():
+    if (
+        request.path != '/login'
```

Comment:
Is it really needed? We need to minimize the checks in `@before_request`, it is expensive.